data.md

```diff
@@ -5,7 +5,7 @@ layout: home
 ---
 
 
-# Accessibility Data
+# Data for Analysis 2025
```

Comment:
The way it is currently set up, this page would just show all the analyses dates, including future dates. Especially if this analysis would only be run once a year, it would not be really sensible to have an 'download by date' option when the page is dated. 

I think we discussed that we could have separate pages per year, so e.g. inscidar.org/data/2025. We could then have a main page inscidar.org/data which would show just the download options, and then each dated page would have the previews I imagine.

Reply:
```suggestion
# Accessibility Data
```

Reply:
It sounds good to have separate per-year pages. Let's revisit this in the future.